src/modules/claims/claims.service.ts

```diff
@@ -1482,10 +1483,22 @@ export class ClaimsService {
     }
     const role = vc.credentialSubject.role.namespace;
     try {
+      let issuerVerificationResult: VerificationResult;
       if (typeof issuerDID === 'string') {
-        await this._issuerVerification.verifyIssuer(issuerDID, role);
+        issuerVerificationResult = await this._issuerVerification.verifyIssuer(
+          issuerDID,
+          role
+        );
       } else {
-        await this._issuerVerification.verifyIssuer(issuerDID.id, role);
+        issuerVerificationResult = await this._issuerVerification.verifyIssuer(
+          issuerDID.id,
+          role
+        );
+      }
+      const { verified, error } = issuerVerificationResult;
+      issuerVerified = issuerVerified && verified;
+      if (error.length > 0) {
+        errors.push(error);
```

Comment:
`push` probably pushes the array and not the elements of the array. Can either use `concat` or spread operator

Reply:
@jrhender It turns out that `error` is a string https://github.com/energywebfoundation/ew-credentials/blob/c9f5790277d09c5ea5c332e15ee15add880f6e6d/packages/vc-verification/src/models/credentials-types.ts#L11

Reply:
Ah okay. I think that `!== ""` is a more idomatic way of checking for an empty string. Would reduce this string/array confusion when reading the code.
```suggestion
      if (error !== "") {
        errors.push(error);
```